github_api_helper.js

```diff
@@ -26,21 +26,37 @@ function inProgressAPIresponse (context, headSha) {
   })
 }
 
+// Filters the files which are not relevant to us
+function filterData (rawData) {
+  return rawData.data.filter(file => config.fileExtensions.reduce((acc, ext) => acc || file.filename.endsWith(ext), false))
+    .filter(fileJSON => fileJSON.status !== 'removed').map(async fileInfo => {
+      return fileInfo.filename
+    })
+}
+
 /**
  * Get list of files modified by a pull request
  * @param {import('probot').Context} context Probot context
  * @param {number} number the pull request number inside the repository
  * @returns {Promise<any>} GitHub response
```

Comment:
Doesn't look like you're returning a GitHub response object any longer.

Reply:
Yes, a good catch.